src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewControl.cs

```diff
@@ -447,7 +447,7 @@ private void ComputePreview()
 
                 document.Print();
                 pageInfo = previewController.GetPreviewPageInfo();
-                Debug.Assert(pageInfo is not null, "ReviewPrintController did not give us preview info");
+                Debug.Assert(pageInfo is not null, "PreviewPrintController did not give us preview info");
```

Comment:
Thank you! Minor suggestion - 
```suggestion
                Debug.Assert(pageInfo is not null, $"{nameof(PreviewPrintController)} did not give us preview info.");
```

Reply:
Applied - should I squash the commits? Or will that be done by the merge?

Thank you!